spec/controllers/api/current/patients_controller_spec.rb

```diff
@@ -149,6 +159,14 @@ def create_record_list(n, options = {})
           expect(db_phone_number.attributes.with_payload_keys.with_int_timestamps)
             .to eq(updated_phone_number)
         end
+
+        expect(PatientBusinessIdentifier.updated_on_server_since(sync_time).count).to eq 10
```

Comment:
This could potentially be refactored by moving nested patient attributes into different specs.

Maybe,
* Scalar values + address as a single test
* Patient Phone number as a separate test
* Business ID as a separate test

Reply:
refactored